docs/_advanced/lazy_listener.md

```diff
@@ -0,0 +1,79 @@
+---
+title: Lazy listeners (FaaS)
+lang: en
+slug: lazy-listeners
+order: 9
+---
+
+<div class="section-content">
+⚠️ Lazy listener functions are a beta feature to make it easier to deploy Bolt for Python apps to FaaS environments. As the feature is developed, Bolt for Python's API is subject to change.
+
+Typically you'd call `ack()` as the first step of your listener functions. Calling `ack()` tells Slack that you've received the event and are handling it in within reasonable amount of time (3 seconds).
+
+However, apps running on FaaS or similar runtimes that don't allow you to run threads or processes after returning an HTTP response cannot follow this pattern. Instead, you should set the `process_before_response` flag to `True`. This allows you to create a listener that calls `ack()` and handles the event in seperate threads or processes, though you still need to complete everything within 3 seconds. For events that do not require acknowledgement, you can create listeners as you normally would.
```

Comment:
How about changing this way? Technically, it uses a thread internally but the more important point here is that process_before_response guarantees the completion of all the processes in a listener. Also, please polish the sentence for Events API. (NOTE: this is not Bolt Python specific - Bolt JS has the same limitation)

```suggestion
However, apps running on FaaS or similar runtimes that don't allow you to run threads or processes after returning an HTTP response cannot follow this pattern. Instead, you should set the `process_before_response` flag to `True`. This allows you to create a listener that calls `ack()` and handles the event safely, though you still need to complete everything within 3 seconds. For events, while a listener doesn't need `ack()` method call as you normally would, the listener needs to complete within 3 seconds, too.
```